README.md

```diff
@@ -16,13 +16,22 @@ npm install --save botmeter-logger
 
 ## How to use
 
-```node.js
+There are 3 ways you can use botmeter-logger:
+
+- generic
+- messenger
+- botbuilder
```

Comment:
At some point we will say there's a fourth way that is using botfuel's bot sdk, and it does not require any custom code. We will add it later in the public docs, so we will here